server/client.go

```diff
@@ -117,6 +117,8 @@ var readLoopReportThreshold = readLoopReport
 // Represent client booleans with a bitmask
 type clientFlag uint16
 
+const hdrLine = "NATS/1.0\r\n"
```

Comment:
We already have this somewhere I thought defined.

Reply:
I don't think so, this is why I moved it here from inside setHeader()

Reply:
I can see it in master. But was scoped inside a function.